doc/rtd/reference/datasources/digitalocean.rst

```diff
@@ -7,10 +7,10 @@ DigitalOcean
 
 
 The `DigitalOcean`_ datasource consumes the content served from DigitalOcean's
-metadata service. This metadata service serves information about the
-running droplet via http over the link local address ``169.254.169.254``. The
-metadata API endpoints are fully described in the DigitalOcean
-`metadata documentation`_.
+instance metadata service. This instance metadata service serves information
+about the running droplet via http over the link local address
+``169.254.169.254``. The meta-data API endpoints are fully described in the
```

Comment:
```suggestion
``169.254.169.254``. The API endpoints are fully described in the
```
Since it fetches more than just `meta-data`, I would just remove the word entirely